include/subgroup/tile/impl/fma_xe.hpp

```diff
@@ -24,6 +24,150 @@
 namespace gpu::xetla::subgroup {
 
 /// @brief Is the tile mma operation functor, specialized for Xe and fpu engine.
+template <
+    typename matAcc_t_,
+    typename matC_t_,
+    typename matB_t_,
+    typename matA_t_,
+    gpu_arch arch_tag_>
+struct tile_fma_t {
```

Comment:
May you consider add a new reg_layout as col_major_tiled, thus make this enabled as:
template <
    typename matAcc_dst_t_,
    typename matAcc_src_t_,
    typename matB_t_,
    typename matA_t_,
    gpu_arch arch_tag_>
struct tile_mma_t<
    matAcc_dst_t_,
    matAcc_src_t_,
    matB_t_,
    matA_t_,
    mma_engine::fpu,
    arch_tag_,
    std::enable_if_t<
        arch_has_fpu<arch_tag_> &&
        matA_t_::register_layout == reg_layout::tiled &&
        matB_t_::register_layout == reg_layout::col_major_tiled>> {
//impl mma for suck kind of reg layout
...
}